keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ArtifactSummary.kt

```diff
@@ -0,0 +1,31 @@
+package com.netflix.spinnaker.keel.core.api
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.netflix.spinnaker.keel.api.artifacts.ArtifactType
+import java.time.Instant
+
+/**
+ * Summarized data about a specific artifact, mostly for use by the UI.
+ */
+data class ArtifactSummary(
+  val name: String,
+  val type: ArtifactType,
+  val versions: Set<ArtifactVersionSummary>?
```

Comment:
should probably not be null, and if there are no versions it should be an empty set